packages/wonder-blocks-data/src/util/request-tracking.js

```diff
@@ -118,37 +118,62 @@ export class RequestTracker {
 
             for (const requestKey of Object.keys(this._trackedRequests)) {
                 const options = this._trackedRequests[requestKey];
-                const hydrate = options.hydrate ?? true;
 
                 try {
                     promises.push(
                         this._requestFulfillment
-                            .fulfill(requestKey, options)
-                            .then((data) => {
-                                if (data == null) {
-                                    // Request aborted. We won't cache this.
-                                    return null;
+                            .fulfill(requestKey, {...options})
```

Comment:
We spread options because the `fulfill` method currently has `hydrate` as optional whereas we assert it can never be null. That messes with flow.